packages/strapi-admin/scripts/setup.js

```diff
@@ -0,0 +1,36 @@
+const shell = require('shelljs');
```

Comment:
Why use new dependency ? You can use `child_process` not ?

Reply:
More easy to use (silent but only display errors) and we are already using it in the repository.